quiche/src/recovery/congestion/mod.rs

```diff
@@ -40,6 +40,16 @@ use crate::StartupExitReason;
 
 pub const PACING_MULTIPLIER: f64 = 1.25;
 
+/// 16 GiB
+#[cfg(any(test, feature = "congestion_window_unchecked_available"))]
+#[cfg(target_pointer_width = "64")]
+pub const UNCHECKED_WINDOW: usize = 16 << 30;
+
+/// 2 GiB
+#[cfg(any(test, feature = "congestion_window_unchecked_available"))]
+#[cfg(not(target_pointer_width = "64"))]
+pub const UNCHECKED_WINDOW: usize = 2 << 30;
```

Comment:
I wonder if it would be preferable to use u64 instead of usize to track this so we can avoid issues related to usize width.